README.md

```diff
@@ -313,6 +313,10 @@ Clone the code and make sure it is on your `$GOPATH`. (Important: we assume in t
 
     $ git clone --recursive https://github.com/hyperledger-labs/fabric-private-chaincode.git $GOPATH/src/github.com/hyperledger-labs/fabric-private-chaincode
 
+Note: above will pull in some large but not needed dependencies from
+PDO. You can omit the corresponding pull by prefixing `clone` with
+`-c submodule.interpreters/wasm-micro-runtime.update=none -c submodule.ccf_transaction_processor/CCF.update=none `.
+
```

Comment:
put this before the clone, so people read it before they do it.

Reply:
> put this before the clone, so people read it before they do it.

Personally I think adding that before is a bit confusing.  If you think most people really would like to use the these options, then we might just add the options to the command right away? (I just didn't do it as i thought not everybody cared and the command would get a bit long/clumsy if all spelled out.  But if the majority of folks would prefer the leaner variant, then it's probably best to just add it to the command right away (and just drop the note altogether?)

Reply:
ufff .... I would not expose this to users here as it makes it too complicated. Isn't there a way to clone with shallow-submodules? 
https://git-scm.com/docs/git-clone#Documentation/git-clone.txt---no-shallow-submodules

Our goal should be to make the fpc `git clone` command as natural/easy as possible. 

If we exclude these submodules in the docker images, that is fine, as long as we dont ask the user to exclude them as well!